event.go

```diff
@@ -0,0 +1,125 @@
+package vlc
+
+// #cgo LDFLAGS: -lvlc
+// #include <vlc/vlc.h>
+// #include <stdlib.h>
+import "C"
+import (
+	"bytes"
+	"encoding/binary"
+)
+
+type EventType uint16
+
+// Event types
+const (
+	MediaMetaChanged EventType = iota
+	MediaSubItemAdded
+	MediaDurationChanged
+	MediaParsedChanged
+	MediaFreed
+	MediaStateChanged
+	MediaSubItemTreeAdded
+)
+
+const (
+	MediaPlayerMediaChanged = iota + 0x100
+	MediaPlayerNothingSpecial
+	MediaPlayerOpening
+	MediaPlayerBuffering
+	MediaPlayerPlaying
+	MediaPlayerPaused
+	MediaPlayerStopped
+	MediaPlayerForward
+	MediaPlayerBackward
+	MediaPlayerEndReached
+	MediaPlayerEncounteredError
+	MediaPlayerTimeChanged
+	MediaPlayerPositionChanged
+	MediaPlayerSeekableChanged
+	MediaPlayerPausableChanged
+	MediaPlayerTitleChanged
+	MediaPlayerSnapshotTaken
+	MediaPlayerLengthChanged
+	MediaPlayerVout
+	MediaPlayerScrambledChanged
+	MediaPlayerESAdded
+	MediaPlayerESDeleted
+	MediaPlayerESSelected
+	MediaPlayerCorked
+	MediaPlayerUncorked
+	MediaPlayerMuted
+	MediaPlayerUnmuted
+	MediaPlayerAudioVolume
+	MediaPlayerAudioDevice
+	MediaPlayerChapterChanged
+)
+
+const (
+	MediaListItemAdded = iota + 0x200
+	MediaListWillAddItem
+	MediaListItemDeleted
+	MediaListWillDeleteItem
+	MediaListEndReached
+)
+
+const (
+	MediaListViewItemAdded = iota + 0x300
+	MediaListViewWillAddItem
+	MediaListViewItemDeleted
+	MediaListViewWillDeleteItem
+)
+
+const (
+	MediaListPlayerPlayed = iota + 0x400
+	MediaListPlayerNextItemSet
```

Comment:
I noticed that you created an `EventManager` for the `player` but not the `listplayer`; is there a reason you didn't implement it for the `listplayer` as well?